### ISSUE TITLE
Grails 4

### ISSUE THREAD

**Issue by tamershahin** [ajay-kmr/swagger#19]
Seems that the plugin is not working with grails 4. is there any plan to support it?

I'll try again maybe with an demo project to investigate better

**Comment by sk-space** [comment]
yes i recently tired to implement it, but not working in grails 4. need some reform here. @ajay-kmr

**Comment by tarekEng** [comment]
Hello @tamershahin, @sk-space did you find any solution to integrate swagger with grails 4 please ? thanks in advance
